### ISSUE TITLE
AgroMo transfer to other location

### ISSUE THREAD

**Issue by fodornandor** [hollorol/AgroMo#114]
Design protocol for using AgroMo for another location.
challenge: what is needed to (how should map creation be enhanced/changed) create gridded maps using query results?

**Comment by hollorol** [comment]
Please give me a **JSON** file, which contains:
1. The bounding box for a region in hungary
2. A soil directory for that region
3. weather directory for that region
4. meteorology file for the region. 
5. The coordinates for center of the cells.

**Comment by fodornandor** [comment]
Is 1. really needed if 5. is provided? The first coordinate of the left-most cell(s) and the second coordinate of the bottom-most cell(s) provide the coordinates of the bottom-left corner of the bounding box... aren't they? The top-right corner can be defined/found similarly. I cannot see the point of 4. Meteorology file for the region? Which file do you mean?

**Comment by hollorol** [comment]
Yes, 1. is not needed, instead you have to provide a resolution.

**Comment by hollorol** [comment]
The meteorological file set...

**Comment by hollorol** [comment]
further more the bounding box can be necessary if you want to add context (if you zoom too closely on a county, you can see the problem)

**Comment by fodornandor** [comment]
> The meteorological file set...

That's 3. (that is why I'm confused... is it repeated?)

**Comment by fodornandor** [comment]
> further more the bounding box can be necessary if you want to add context (if you zoom too closely on a county, you can see the problem)

The original bounding box info can be retreived from the cell lat/long data... I cannot see why/how zooming in causes a problem ... maybe zoom out?

**Comment by hollorol** [comment]
If you want to see Hungary for example you have some outer region parts... With this, you can see the surrounding countries (partly) as well. As I remember in the first version, we only ploted just the points... You said that plot some part of the surrounding also...

**Comment by hollorol** [comment]
> > The meteorological file set...
> 
> That's 3. (that is why I'm confused... is it repeated?)

Yes, it was accidentally. I changed it to soil

**Comment by fodornandor** [comment]
> > > The meteorological file set...
> > 
> > 
> > That's 3. (that is why I'm confused... is it repeated?)
> 
> Yes, it was accidentally. I changed it to soil

That is 2. :)

**Comment by fodornandor** [comment]
> If you want to see Hungary for example you have some outer region parts... With this, you can see the surrounding countries (partly) as well. As I remember in the first version, we only ploted just the points... You said that plot some part of the surrounding also...

OK. So that is zoom out... I think it's not necesseraly a feature of the original grid, in stead this info belongs to the zooming process.

**Comment by hollorol** [comment]
That is a new feature... In the current process. We control it with bounding box, and there is no zoom.

**Comment by fodornandor** [comment]
OK, then I would minimize the margin around the area we'd like to focus on... but I can let it go and include bound box info as well.

**Comment by fodornandor** [comment]
I think we need only these:
1) bottom-left & top-right corners of the bounding box (lat/long in harmony with the cell coordinates)
2) resolution
3) lat/lon of the grid cells (of the region)

the soil and weather file folders come with the story file, here we just define the grid
